hive-metastore/src/test/resources/hive-schema-3.1.0.derby.sql

```diff
@@ -130,7 +130,7 @@ CREATE TABLE "APP"."TAB_COL_STATS"(
     "BIT_VECTOR" BLOB
 );
 
-CREATE TABLE "APP"."TABLE_PARAMS" ("TBL_ID" BIGINT NOT NULL, "PARAM_KEY" VARCHAR(256) NOT NULL, "PARAM_VALUE" CLOB);
```

Comment:
Do we want to change the other `CLOB`s in this file as well?

Reply:
Add more changes to be consistent with HIVE-16667 and HIVE-25574

Reply:
FYI CLOB handling was fixed in https://github.com/apache/hive/pull/5386/files#diff-bcca13f6cc251df321e8fe80568ef0334a1d44f7e5e7ff2fcaa06ab4f05bbdf9R3387

Reply:
Good to know the direct SQL is made more robust. Maybe we can revisit this change when we upgrade our hive dependency.